content/features/featuresDeepDive/mesh/transforms/parent_pivot.md

```diff
@@ -10,9 +10,9 @@ video-content:
 
 ## Transformation, Pivots and Parents
 
-Should you wish to rotate or scale a mesh, or other object, about a point other than its own _local origin_ then this can be done by changing its center of transformation using either a parent or a pivot. Whilst in general conversation, and some 3D applications, a center of transformation may be referred to as a pivot point, in Babylon.js the term _pivot_ is used when setting a center of transformation with a specific method. The effects of using a parent or a pivot are different. To obtain the behavior of a pivot point as described for example by 3DS Max or Maya then set a parent on the mesh to act as the pivot point rather than using _setPivotPoint_.

```

Comment:
these changes I meant to send in a separate PR, but it's small enough and hopefully makes sense, so i'll leave em here